### PR TITLE
[LSP6]: modify permissions orders

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

- The position/order of all the permissions defined in LSP6 have been changed.
- All the SUPER permissions are now present on the left side of the bit range, and **mirror** (left to right) their normal permission counterpart.

```
CHANGE OWNER
ADD PERMISSIONS
CHANGE PERMISSIONS
TRANSFER VALUE
CALL
STATIC CALL
DELEGATE CALL
DEPLOY
SET DATA
ENCRYPT
DECRYPT
SIGN
...


S SET DATA
S DELEGATE CALL
S STATIC CALL
S CALL
S TRANSFER VALUE
```

permissions

### CODE REVIEW

File: LSPs/LSP-6-KeyManager.md [hunk 1]
```diff
@@ -194,35 +194,35 @@ DELEGATECALL        = 0x00000000000000000000000000000000000000000000000000000000
...
-ENCRYPT             = 0x0000000000000000000000000000000000000000000000000000000000000400
-// [....] .... 1000 .... ....
+ENCRYPT                = 0x0000000000000000000000000000000000000000000000000000000000000200;
+// [....] .... 0100 .... ....
 // Allows decrypt messages on behalf of the controlled contract, for example for private messages
-DECRYPT             = 0x0000000000000000000000000000000000000000000000000000000000000800
+DECRYPT             = 0x0000000000000000000000000000000000000000000000000000000000000400
+// [....] .... 1000 .... ....
+// Allows signing on behalf of the controlled contract, for example for login purposes
+SIGN             = 0x0000000000000000000000000000000000000000000000000000000000000800
```
Comment (JeneaVranceanu) [inline LSPs/LSP-6-KeyManager.md:207]: Looks like the spacing went off a bit.

Снимок экрана 2022-09-21 в 10 43 24

File: LSPs/LSP-6-KeyManager.md [hunk 2]
```diff
@@ -194,35 +194,35 @@ DELEGATECALL        = 0x00000000000000000000000000000000000000000000000000000000
...
-// [....] .... 0100 .... ....
 // Allows encrypting messages on behalf of the controlled contract, for example for private messages
-ENCRYPT             = 0x0000000000000000000000000000000000000000000000000000000000000400
-// [....] .... 1000 .... ....
+ENCRYPT                = 0x0000000000000000000000000000000000000000000000000000000000000200;
+// [....] .... 0100 .... ....
 // Allows decrypt messages on behalf of the controlled contract, for example for private messages
-DECRYPT             = 0x0000000000000000000000000000000000000000000000000000000000000800
+DECRYPT             = 0x0000000000000000000000000000000000000000000000000000000000000400
+// [....] .... 1000 .... ....
```
Comment (JeneaVranceanu) [inline LSPs/LSP-6-KeyManager.md:205]: To be honest I'm not sure how useful that will be in the long run to have and maintain binary representation.
I removed this part from iOS library.

Comment (JeneaVranceanu) [reply LSPs/LSP-6-KeyManager.md:205]: Especially considering that now we have permissions on both ends of the permission value.

**Review by JeneaVranceanu — CHANGES_REQUESTED** [review]

File: LSPs/LSP-6-KeyManager.md (hunk 2 above)
Comment (CJ42) [reply LSPs/LSP-6-KeyManager.md:205]: I agree let's remove the bits in comments

**Review by frozeman — APPROVED** [review]

**Pull request merged**